### ISSUE TITLE
GPU utilization rate and distribution training

### ISSUE THREAD

**Issue by byrsongyuxin** [kenshohara/3D-ResNets-PyTorch#245]
when I use distribution training on GPUs,  the process ids are the same and the GPU utilization rate is zero.
![rate](https://user-images.githubusercontent.com/42733639/112492954-b1b92100-8dbc-11eb-92d4-b1bf12542078.png)
![pid](https://user-images.githubusercontent.com/42733639/112492960-b2ea4e00-8dbc-11eb-8e22-e720dd207eaa.png)

**Comment by byrsongyuxin** [comment]
CUDA_VISIBLE_DEVICES=0,1 python main.py \
--root_path /dssg/syx/action-recognition/data \
--video_path /dssg/syx/action-recognition/data/UCF101/UCF-101-jpg \
--annotation_path /dssg/syx/action-recognition/data/UCF101/ucf101_01.json \
--result_path /home/wuming/syx/action-recognition/3D-ResNets-PyTorch-master/result \
--dataset ucf101 --n_classes 101 --n_pretrain_classes 700 \
--pretrain_path /home/wuming/syx/action-recognition/3D-ResNets-PyTorch-master/pretrain/r3d50_K_200ep.pth \
--ft_begin_module fc \
--model resnet --model_depth 50 \
--batch_size 128 --n_threads 4 --checkpoint 5 \
--batchnorm_sync --distributed
